### PR TITLE
Include Interlis drivers in windows build

### PR DESCRIPTION
**Pull request by dersteppenwolf**
Fix for https://github.com/conda-forge/gdal-feedstock/issues/398

### CODE REVIEW

**Comment by ocefpaf** [comment]
@dersteppenwolf please bump the build number to create a new package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
     - disable_jpeg12.patch  # [win]
 
 build:
-  number: 3
+  number: 4
```
Comment (ocefpaf) [inline recipe/meta.yaml:15]: :tada:

File: recipe/set_bld_opts.bat [hunk 2]
```diff
@@ -45,6 +45,7 @@ set BLD_OPTS=%WIN64% ^
     HDF5_DIR=%LIBRARY_PREFIX% ^
     HDF5_LIB=%LIBRARY_LIB%\hdf5.lib ^
     HDF5_H5_IS_DLL=YES ^
+    ILI_ENABLED=YES ^
```
Comment (ocefpaf) [inline recipe/set_bld_opts.bat:48]: I don't know the first thing about this driver but we should probably add a test for it.

Comment (dersteppenwolf) [reply recipe/set_bld_opts.bat:48]: Colombia is using it as a "standard"  https://github.com/SwissTierrasColombia/LADM_COL

Comment (ocefpaf) [reply recipe/set_bld_opts.bat:48]: If you can come up with some data to test it or check the current tests we have: https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/run_test.py

Comment (dersteppenwolf) [reply recipe/set_bld_opts.bat:48]: looks like there are some public examples here https://www.interlis.ch/en/downloads/testdatensaetze

Comment (ocefpaf) [reply recipe/set_bld_opts.bat:48]: Cool! And the files are small enough to just use them here. Can you add that test?

**Comment by xylar** [comment]
@dersteppenwolf, are you still interested in having this feature, and are you able to add the tests discussed above?  Some conflicts will have to be resolved as well, since several other PRs have been merged (or will be merged) in the meantime.

**Comment by xylar** [comment]
I'm going to close this for lack of activity.  Feel free to re-open and add the requested tests.

**Pull request closed without merging**